client/src/components/ChannelInfo/index.js

```diff
@@ -0,0 +1,156 @@
+    id: `channel-info-tab-${index}`,
+    "aria-controls": `channel-info-tabpanel-${index}`,
+  };
+}
+
+export default function ChannelInfo(props) {
+  const [repoInfo, setRepoInfo] = useState({});
+  const [isPrivate, setIsPrivate] = useState(false);
+
+  useEffect(() => {
+    const ghRepoInfo = async () => {
+      setIsPrivate(false);
+      try {
+        // Fetches repository information
+        const repository = props.location.pathname
+          .split("/")[2]
+          .replace("_", "/");
+        const headers = {
+          accept: "application/json",
+        };
+        if (Cookies.get("gh_private_repo_token")) {
+          headers["Authorization"] = `token ${Cookies.get(
+            "gh_private_repo_token"
+          )}`;
+        }
+        const ghRepoInfoResponse = await axios({
+          method: "get",
+          url: `${githubApiDomain}/repos/${repository}`,
+          headers: headers,
+        });
+        setRepoInfo(ghRepoInfoResponse.data);
+      } catch (error) {
+        console.log(error);
+        setIsPrivate(true);
+      }
+    };
+    ghRepoInfo();
+  }, [props.location.pathname]);
+
+  const [activeTab, setActiveTab] = useState(0);
+  const repoURL = `https://github.com/${props.location.pathname
+    .split("/")[2]
+    .replace("_", "/")}`;
+
+  const handleChange = (event, newValue) => {
+    setActiveTab(newValue);
+  };
+
+  return (
+    <div className="channel-info-wrapper">
+      <Tabs
+        value={activeTab}
+        onChange={handleChange}
+        aria-label="channel info tabs"
+        indicatorColor="none"
+      >
+        <Tab
+          className={`channel-info-tab ${activeTab == 0 ? "active-tab" : ""}`}
+          label="People"
+          {...a11yProps(0)}
+        />
+        <Tab
+          className={`channel-info-tab ${activeTab == 1 ? "active-tab" : ""}`}
+          label="Repo Info"
+          {...a11yProps(1)}
+        />
+      </Tabs>
+      <TabPanel value={activeTab} index={0}>
+        {/* TODO: Users Online */}
+        Users
+      </TabPanel>
+      <TabPanel value={activeTab} index={1}>
+        <div className="repo-info-wrapper">
+          {!isPrivate ? (
+            <>
+              <div className="repo-info-header">
+                <span>{repoInfo.name} </span>
+              </div>
+              <div className="repo-info-description">
+                <div>
+                  <span>{repoInfo.description}</span>
+                </div>
+                <div>
+                  <span>
+                    <i>{repoInfo.language}</i>
+                  </span>
+                </div>
+                <div style={{ marginTop: "10px" }}>
+                  <span>by {repoInfo.owner ? repoInfo.owner.login : ""}</span>
+                </div>
+              </div>
+
+              <div className="repo-info-stats">
+                <span>
+                  <a href={`${repoURL}/issues`}>
+                    <strong>{repoInfo.open_issues_count}</strong> issues
```

Comment:
This gives us PRs + Issues, and the link redirects us to the issues. There must be another field to gives the issues only number.

Reply:
It seems like there isn't. This is because GitHub treats both issues and PRs as issues. Knowing the number of issues would require a call to the `/issues` endpoint for a repository which also includes the PRs within. The PRs are distinguishable by the presence of a `pull_request` key.

Reference - https://docs.github.com/en/rest/reference/issues#list-repository-issues